tests/utils/stl/test/tests.py

```diff
@@ -34,12 +34,13 @@ def __init__(self, suite, path_in_suite, lit_config, test_config,
 
         self._configure_cxx(lit_config, envlst_entry, default_cxx)
 
-        # TRANSITION: These configurations should be enabled in the future.
         for flag in chain(self.cxx.flags, self.cxx.compile_flags):
-            if flag.startswith('clr:pure', 1):
-                self.requires.append('clr_pure')
-            elif flag.startswith('BE', 1):
-                self.requires.append('edg')
+            if flag[1:] == 'clr:pure':
+                self.requires.append('clr_pure') # TRANSITION, GH-798
+            elif flag[1:] == 'clr':
+                self.requires.append('clr') # TRANSITION, GH-797
+            elif flag[1:] == 'BE':
+                self.requires.append('edg') # TRANSITION, GH-799

```

Comment:
As I noted in #799 this doesn't actually disable the `/BE` tests. It just adds a requirement to any test which uses the `/BE` flag. If we're targeting x86 this gets enabled in `tests/utils/stl/test/config.py`.

Reply:
So the pre-existing `TRANSITION` was doubly bad.

Reply:
Oh, we actually have EDG x86 coverage? I got confused because I've only been building and testing x64.

Reply:
Fixed comment, thanks!